### PR TITLE
Fix wrong connection name in Druid Hook

### PR DESCRIPTION
**Pull request by potiuk**
The connection attribute name was wrong

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

### CODE REVIEW

**Comment by potiuk** [comment]
Last one to release providers :)

File: airflow/providers/apache/druid/hooks/druid.py [hunk 1]
```diff
@@ -136,7 +136,7 @@ class DruidDbApiHook(DbApiHook):
     For ingestion, please use druidHook.
     """
 
-    conn_name_attr = 'druid_broker_conn_id'
+    conn_name_attr = 'conn_id'
```
Comment (kaxil) [inline airflow/providers/apache/druid/hooks/druid.py:140]: Can you explain this change a bit

Comment (kaxil) [reply airflow/providers/apache/druid/hooks/druid.py:140]: I don't think this is a wrong connection name -- we use the same everywhere for other hooks too

Comment (potiuk) [reply airflow/providers/apache/druid/hooks/druid.py:140]: Right, I close that one and take a closer look.

**Comment by potiuk** [comment]
Yeah. It was false alarm. All works fine for my original implementation - just double checked it. I think it was an error on user side (workers were still using old provider) and I will ask the user to try it again when I release rc2

**Pull request closed without merging**